inception-project/inception
# #2599 - Allow a default sidebar per project be open

**What's in the PR**
- Added a new project settings page for "Annotation" and there the project manager can configure a default sidebar
- Added a description to sidebar factories
- Added available(...) method allowing to check if a sidebar factory in general is available in the current project or not
- Added option to use a default preference which is used as the fallback for a user preference if the user does not have a preference yet

**How to test manually**
* Try setting a default sidebar. Note that one a user opens the annotation page, a sidebar preference is saved and this means the user won't fall back to the default preference. So either you need to try with a new user, a new project or you'd have to delete the user preference from the DB to test the fallback.

**Automatic testing**

**Documentation**
